Resources/Prototypes/Entities/Structures/Storage/Closets/Lockers/lockers.yml

```diff
@@ -327,7 +326,7 @@
 - type: entity
   id: LockerSecurity
   parent: LockerBaseSecure
-  name: deputy's locker
+  name: security officer's locker
```

Comment:
you're bulldozing other changes here, make sure to keep the proper entries for these